test_data/swc/non_increasing_trunk_off_1_16pt.swc

```diff
@@ -27,21 +27,21 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Example SWC file with non-increasing indices.
-# This should fail to load early.
+# 6 and 12 are swapped w/ 7 and 13
 #
-1 0 0 0 0 0 -1
+1 1 0 0 0 0 -1
```

Comment:
second column shouldn't be increasing: causes the soma detection to get confused as there are multiple soma segments